saleor/graphql/invoice/types.py

```diff
@@ -0,0 +1,22 @@
+import graphene
+
+from ...invoice import models
+from ..core.connection import CountableDjangoObjectType
+from ..core.types.common import Job
+from ..meta.types import ObjectWithMetadata
+
+
+class Invoice(CountableDjangoObjectType):
+    url = graphene.String(description="URL to download an invoice.")
+
+    class Meta:
+        description = "Represents an Invoice."
+        interfaces = [ObjectWithMetadata, Job]
```

Comment:
Can you check if Invoice resolver returns ID as graphene ID instead of PK?
If yes we should add: 
```suggestion
        interfaces = [ObjectWithMetadata, Job, relay.Node]
```